lib/node_modules/@stdlib/stats/base/range/docs/repl.txt

```diff
@@ -1,92 +1,124 @@
+{{alias}}( N, x, strideX )
+
 
-{{alias}}( N, x, stride )
     Computes the range of a strided array.
 
-    The `N` and `stride` parameters determine which elements in `x` are accessed
-    at runtime.
 
-    Indexing is relative to the first index. To introduce an offset, use a typed
-    array view.
+    The `N` and stride parameters determine which elements in the strided 
+    array are accessed at runtime.
+
+
+    Indexing is relative to the first index. To introduce an offset, use a 
+    typed array view.
+
 
     If `N <= 0`, the function returns `NaN`.
 
+
     Parameters
     ----------
+
+
     N: integer
         Number of indexed elements.
 
+
     x: Array<number>|TypedArray
         Input array.
 
-    stride: integer
-        Index increment.
+
+    strideX: integer
+        Stride length.
+
 
     Returns
     -------
+
+
     out: number
         Range.
 
+
     Examples
     --------
+
+
     // Standard Usage:
     > var x = [ 1.0, -2.0, 2.0 ];
     > {{alias}}( x.length, x, 1 )
     4.0
 
-    // Using `N` and `stride` parameters:
+
```

Comment:
You need to go through and remove all these stray lines. Please study other repl.txt files.